pkg/server/server.go

```diff
@@ -90,6 +93,62 @@ func (s *Server) SetRules(w http.ResponseWriter, r *http.Request, tenant string)
 	_, _ = w.Write([]byte("successfully updated rules file"))
 }
 
+func (s *Server) ListAllRules(w http.ResponseWriter, r *http.Request) {
+	logger := log.With(s.logger, "handler", "listAllRules")
+
+	//nolint:exhaustivestruct
+	allGroups := &rulefmt.RuleGroups{}
+
+	_ = s.bucket.Iter(r.Context(), rulesBasePath, func(dir string) error {
```

Comment:
Since we don't use the returned values, I wonder if we can more concisely say:
```suggestion
	s.bucket.Iter(r.Context(), rulesBasePath, func(dir string) error {
```

Reply:
Ah nvm, it's an error thay we are explicitly ignoring